shared-helpers/__tests__/pdfs.test.ts

```diff
@@ -8,6 +9,10 @@ describe("pdfs helpers", () => {
   const OLD_ENV = process.env
   process.env.cloudinaryCloudName = "exygy"
 
+  beforeAll(() => {
+    FileServiceProvider.create()
```

Comment:
So at a higher level I think this (doing FileServiceProvider.create() at app startup) seems fine however am I right that this is a little different than the approach that you outlined in the CL? I think the OG plan was to build this at runtime (and then I suggested some stuff on validating the env variables in `next.config.js` at startup)